raspberry/anemometer/read_trisonica_mini_data_rolling.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import serial
+import numpy as np
+
+all_data = []
+Wind_Speed = []
+Wind_Speed_2D = []
+H_direction =[]
+V_direction = []
+U_axis = []
+V_axis = []
+W_axis = []
+
+for x in range(20):
+   Wind_Speed.append(0.00)
+   Wind_Speed_2D.append(0.00)
+   H_direction.append(0.00)
+   V_direction.append(0.00)
+   U_axis.append(0.00)
+   V_axis.append(0.00)
+   W_axis.append(0.00)
+
+if __name__ == '__main__':
+    ser = serial.Serial('/dev/ttyS0', 115200, timeout=1)
+    ser.reset_input_buffer()
+
+    while True:
+        if ser.in_waiting > 0:
+            line = ser.readline().decode('utf-8').rstrip()
+            print(line)
+
+            all_data = line.split()
+
+            Wind_Speed = np.roll(Wind_Speed, 1)
```

Comment:
You don't need roll even if you want to keep all the data in a buffer. But is that even necessary?